com.unity.render-pipelines.universal/Editor/ForwardRendererDataEditor.cs

```diff
@@ -16,7 +16,7 @@ private static class Styles
             public static readonly GUIContent OpaqueMask = new GUIContent("Opaque Layer Mask", "Controls which opaque layers this renderer draws.");
             public static readonly GUIContent TransparentMask = new GUIContent("Transparent Layer Mask", "Controls which transparent layers this renderer draws.");
             public static readonly GUIContent LightingLabel = new GUIContent("Lighting", "Controls how lights are shaded.");
-            public static readonly GUIContent DeferredShadingLabel = new GUIContent("Deferred Shading", "Enable deferred shading on compatible lights.");
+            public static readonly GUIContent ShadingLabel = new GUIContent("Shading", "Enable deferred shading on compatible lights.");
```

Comment:
text need to be updated to explain the options.

Reply:
I renamed to RenderingMode to match built-in.